examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,39 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
+:class:`xarray.DataArray` format, and plot it on a map.
+
+The example below shows a Worldview 2 satellite image over
+`Lāhainā, Hawaiʻi during the August 2023 wildfires
+<https://en.wikipedia.org/wiki/2023_Hawaii_wildfires#L%C4%81hain%C4%81>`_.
+Data is sourced from a Cloud-Optimized GeoTIFF (COG) file hosted on
+`OpenAerialMap <https://map.openaerialmap.org>`_ under a
+`CC BY-NC 4.0 <https://creativecommons.org/licenses/by-nc/4.0/>`_ license.
+"""
+import pygmt
+import rioxarray
+
+###############################################################################
+# Read 3-band data from GeoTIFF into an xarray.DataArray object
+with rioxarray.open_rasterio(
+    filename="https://oin-hotosm.s3.us-east-1.amazonaws.com/64d6a49a19cb3a000147a65b/0/64d6a49a19cb3a000147a65c.tif",
+    overview_level=5,
+) as img:
+    # Subset to area of Lāhainā in EPSG:32604 coordinates
+    image = img.rio.clip_box(minx=738000, maxx=755000, miny=2300000, maxy=2318000)
+    image = image.load()  # force loading dataarray into memory
```

Comment:
> Note that the docs build for this example segfaults on both Readthedocs and Ubuntu, see e.g. https://github.com/GenericMappingTools/pygmt/actions/runs/5972759386/job/16203804119?pr=2641#step:7:82. Running this locally, I actually get a segfault too on the first run, but running `cd doc && make all` a second time works. Not quite sure what the problem is.

Running `.load()` should fix the segfault? See https://github.com/corteva/rioxarray/issues/550#issuecomment-1216690984, and we actually use `dataarray.load()` in the `pygmt.load_dataarray()` function in #1439.

Reply:
Yep, it works! See https://pygmt-dev--2641.org.readthedocs.build/en/2641/gallery/images/rgb_image.html